qupath-gui-fx/src/main/java/qupath/lib/display/ImageDisplay.java

```diff
@@ -705,7 +705,7 @@ public Histogram getHistogram(ChannelDisplayInfo info) {
 	 * 
 	 * @return
 	 */
-	private String toJSON() {
+	public String toJSON( ) {
```

Comment:
So that we can set the data into a text file for reuse, very useful when going across projects
https://github.com/BIOP/qupath-biop-extensions/blob/dev-0.2.0/src/main/java/ch/epfl/biop/qupath/commands/LoadDisplaySettingsCommand.java